### ISSUE TITLE
Fix: During page load, the text remains visible

### ISSUE THREAD

**Issue by akhanfir** [Meeds-io/meeds#414]
**Steps to reproduce**

1. In page Stream open lighthouse

**(optional) Affected version**

This affects all versions Meeds until 1.4.x.

**Current behaviour**

the lighthouse displays this problem (During the webfont load, text is not visible).

**Expected behavior**

there is no such problem.
